source/expressions.tex

```diff
@@ -5238,6 +5238,11 @@
 the result denotes that object or function;
 otherwise, the behavior is undefined except as specified in \ref{expr.typeid}.
 \begin{note}
+Indirection through a pointer to an object outside that object's
+lifetime is valid. The lvalue thus obtained can be used in limited
+ways, see~\ref{basic.life}.
```

Comment:
I think we don't have to copy the other text verbatim, and we don't have to go into details about "limited ways".  How about:

```suggestion
Indirection through a pointer to an out-of-lifetime object is valid\iref{basic.life}.
```